dashboard-widgets.el

```diff
@@ -438,6 +450,8 @@ If MESSAGEBUF is not nil then MSG is also written in message buffer."
     (insert " "))
 
   (insert (propertize heading 'face 'dashboard-heading))
+  (overlay-put (make-overlay (- (point) (length heading)) (point))
+               'display (or (cdr (assoc heading dashboard-item-names)) heading))
```

Comment:
You might want to preserve the face, and this does not work when `dashboard-center-content` is set to `t`. Make sure to pass in `t` to make-overlay's fourth argument FRONT-ADVANCE so the insertion will be covered.

hence,

```el
;; Some comment here...
(let ((ov (make-overlay (- (point) (length heading)) (point) nil t)))
    (overlay-put ov 'display (or (cdr (assoc heading dashboard-item-names)) heading))
    (overlay-put ov 'face 'dashboard-heading))
```

Can you also add some comment about the changes?

Reply:
@jcs090218 ,

I applied your changes and added a comment.  Thank you for the feedback.

>You might want to preserve the face
During my (admittedly brief) tests I found that the original face was already
preserved since the overlay didn't set one to override it.
But I agree, it's better to explicitly apply the face again just to make sure.

Reply:
> During my (admittedly brief) tests I found that the original face was already
preserved

I think  you are right, but it does not preserved on my machine. This seems strange. 😕